test/integration/vaultstaticsecret_integration_test.go

```diff
@@ -367,6 +366,19 @@ func TestVaultStaticSecret_kv(t *testing.T) {
 			} else {
 				assertNoHMAC(t, obj)
 			}
+
+			if obj.Spec.Destination.Create {
+				sec, _, err := helpers.GetSecret(ctx, crdClient, obj)
+				if assert.NoError(t, err) {
+					// ensure that a Secret deleted ouf of band is properly restored
```

Comment:
Nitpick: 
```suggestion
					// ensure that a Secret deleted out of band is properly restored
```

Reply:
Fixed in https://github.com/hashicorp/vault-secrets-operator/pull/137/commits/e3bda36c3b3443478109248c200fe0926896a9a5